ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_protection_schedule.py

```diff
@@ -0,0 +1,510 @@
+      that has application synchronization. The tool used to verify snapshot depends on the type of application. For example, if application
+      synchronization is VSS and the application ID is Exchange, eseutil tool is run on the snapshots. If verification fails, the logs are not truncated.
+  state:
+    required: True
+    choices:
+        - present
+        - absent
+        - create
+    type: str
+    description:
+    - Choice for protection schedule operations
+  until_time:
+    required: False
+    type: int
+    default: 86399
+    description:
+    - Time of day to stop taking snapshots. Applicable only when repeat frequency specifies more than one snapshot in a day.
+  use_downstream_for_DR:
+    required: False
+    type: bool
+    description:
+    - Break synchronous replication for the specified volume collection and present downstream volumes to host(s). Downstream volumes in the volume
+      collection will be set to online and presented to the host(s) using new serial and LUN numbers. No changes will be made to the upstream volumes,
+      their serial and LUN numbers, and their online state. The existing ACLs on the upstream volumes will be copied to the downstream volumes.
+      Use this in conjunction with an empty downstream_partner_id. This unconfigures synchronous replication when the partner is removed from the
+      last replicating schedule in the specified volume collection and presents the downstream volumes to host(s). Host(s) will need to be configured
+      to access the new volumes with the newly assigned serial and LUN numbers. Use this option to expose downstream volumes in a synchronously replicated
+      volume collection to host(s) only when the upstream partner is confirmed to be down and there is no communication between partners. Do not execute this
+      operation if a previous Group Management Service takeover has been performed on a different array. Do not perform a subsequent Group Management Service
+      takeover on a different array as it will lead to irreconcilable conflicts. This limitation is cleared once the Group management service backup array has
+      successfully synchronized after reconnection.
+  volcoll_or_prottmpl_type:
+      choices:
+        - protection_template
+        - volume_collection
+    required: False
+    type: str
+    description:
+    - Type of the protection policy this schedule is attached to.
+  volcoll_name:
+    required: False
+    type: str
+    description:
+    - Name of the volume collection in which this protection schedule is attached to.
+extends_documentation_fragment: hpe_nimble
+short_description: Manage HPE Nimble Storage protection schedule.
+version_added: 2.9
+'''
+
+EXAMPLES = r'''
+
+# if state is create , then create a protection schedule if not present. Fails if already present.
+# if state is "present", then create a protection schedule if not present. Succeed if it already exists.
```

Comment:
Why is present in quotes but create isn't above?

Reply:
removed